src/validators/object.ts

```diff
@@ -129,20 +129,52 @@ export abstract class ObjectValidator<T extends ObjectSchema = never, O = never>
   }
 
   public toString(options?: ValidatorExportOptions): string {
+    if (options?.types === true) {
+      return this.typeString(options)
+    } else {
+      return this.constructorString(options)
+    }
+  }
+
+  protected validateValue(value: unknown, context?: string, options?: ValidateOptions): ValidationFailure[] {
+    return validateObject(this.schema, value, context, { earlyFail: this.earlyFail, ...options })
+  }
+
+  private typeString(options?: ValidatorExportOptions): string {
+    const language = options?.language ?? 'typescript'
+    switch (language) {
+      case 'typescript': {
+        const lines = Object.keys(this.schema).map(k =>
+          `'${k}': ${this.schema[k].toString(options)}`.replace(/(^|\n)/g, '$1  ')
+        )
+        let typeStr = `{\n${lines.join('\n')}\n}`
```

Comment:
To make it really nice you could handle the case where the object is empty {} and don't add the new lines.

fx.

``` typescript
let typeStr =  lines === '' ? `{}` : `{\n${lines.join('\n')}\n}`
```

Reply:
good idea